Packs/EDL/ReleaseNotes/2_1_4.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Palo Alto Networks PAN-OS EDL Service
+- Fixed an issue where **test-modules** was failing when "Indicators Query" parameter wasn't supplied.
```

Comment:
**test-module** - not plural.
In any case, I think it'll be more understandable if you say "where clicking on the **Test** button was ..."